packages/ui/src/views/repo/components/branch-selector/branch-selector-dropdown.tsx

```diff
@@ -59,14 +52,15 @@ export const BranchSelectorDropdown: FC<BranchSelectorDropdownProps> = ({
         ) : (
           <span className="text-14 font-medium leading-none">Switch branches/tags</span>
         )}
-
-        <SearchBox.Root
-          className="mt-2 w-full"
-          placeholder={BRANCH_SELECTOR_LABELS[activeTab].searchPlaceholder}
-          value={searchQuery}
-          handleChange={handleSearchChange}
-          showOnFocus
-        />
+        <div role="presentation" onKeyDown={e => e.stopPropagation()}>
```

Comment:
why are we adding this?

Reply:
branch selector takes the focus to DropdownMenuItems as soon as we enter something in the searchbox, so just to trap focus to searchbox till we are typing.